features/YellowBox.feature

```diff
@@ -4,7 +4,79 @@ Feature: Managing Features with the YellowBox
   as a customer
   I want to be able approve and decline features
 
+  Background:
+    Given the store operates on a single channel in "EUR" currency
+
   @ui
   Scenario: Visit the shop-page
     When I am on the yellow-box page
```

Comment:
What exactly do you mean with yellow-box page? Isn't the yellow box displayed on every page?

Reply:
Yes it is yellow-box page is just an alias for the start page.